src/features/OntologyTermCreateGeneralBehavior.feature

```diff
@@ -137,3 +137,24 @@ Feature: Ontology Term Create - General Behavior
         When user selects 'Save' button on ontology list page
         Then user can see "Description must be less than 30 characters." below the 'Method Description' field on ontology list page
         Then user can see banner appears with an error message "Fix Invalid Fields"
+
+    @debug
+    @BI-1698
+    Scenario: Ontology term types
+        Given user selects 'New Term' button on ontology list page
+        Then user can see 'Term Type' dropdown on ontology list page
+        Then user can see "Phenotype" as option of Term Type dropdown on ontology list page
+        Then user can see "Germplasm Attribute" as option of Term Type dropdown on ontology list page
+        Then user can see "Germplasm Passport" as option of Term Type dropdown on ontology list page
+        When user selects "Import Data" in navigation
+        And user selects "Ontology" tab
+        Then user can header "Import File"
```

Comment:
```suggestion
        Then user can see header "Import File"
```

I recommend fixing this. It's defined [here in steps.js](https://github.com/Breeding-Insight/taf/blob/38d447aa91fda71f6fc6d871162434917b9ea372/src/step_definitions/steps.js#L806) and used one other place, [in OntologyImportBreeder.feature](https://github.com/Breeding-Insight/taf/blob/38d447aa91fda71f6fc6d871162434917b9ea372/src/features/OntologyImportBreeder.feature#L13), so it will need to be changed in all 3 places.

Reply:
Done.